src/libp2p/discovery.rs

```diff
@@ -609,17 +610,23 @@ mod tests {
 
     #[tokio::test]
     async fn resolve_libp2p_dnsaddr_test() {
-        let addr = Multiaddr::from_str("/dnsaddr/bootstrap.butterfly.fildev.network").unwrap();
-        let p = addr
-            .iter()
-            .find(|p| matches!(p, Protocol::Dnsaddr(_)))
-            .unwrap();
-        if let Protocol::Dnsaddr(name) = p {
-            let pairs = resolve_libp2p_dnsaddr(&name).await.unwrap();
-            assert!(!pairs.is_empty());
-        } else {
-            panic!("No dnsaddr protocol found");
+        async fn run() -> anyhow::Result<()> {
+            let addr = Multiaddr::from_str("/dnsaddr/bootstrap.butterfly.fildev.network").unwrap();
```

Comment:
```suggestion
            let addr = Multiaddr::from_str("/dnsaddr/bootstrap.libp2p.io").unwrap();
```

Reply:
(probably the domain is more stable than `fildev`)

Reply:
@LesnyRumcajs  Done. Please re-approve.